App crash when starting overlay mode from main activity

The following message is seen:

```
FATAL EXCEPTION: main
    Process: com.guillermonegrete.tts, PID: 6258
    java.lang.RuntimeException: Unable to start service com.guillermonegrete.tts.services.ScreenTextService@bae7c4c with Intent { act=startService cmp=com.guillermonegrete.tts/.services.ScreenTextService launchParam=MultiScreenLaunchParams { mDisplayId=0 mBaseDisplayId=0 mFlags=0 } (has extras) }: java.lang.NullPointerException: Parameter specified as non-null is null: method kotlin.jvm.internal.Intrinsics.checkNotNullParameter, parameter mainThread
        at android.app.ActivityThread.handleServiceArgs(ActivityThread.java:3787)
        at android.app.ActivityThread.-wrap23(ActivityThread.java)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1764)
        at android.os.Handler.dispatchMessage(Handler.java:102)
        at android.os.Looper.loop(Looper.java:154)
        at android.app.ActivityThread.main(ActivityThread.java:6816)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1563)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1451)
     Caused by: java.lang.NullPointerException: Parameter specified as non-null is null: method kotlin.jvm.internal.Intrinsics.checkNotNullParameter, parameter mainThread
        at com.guillermonegrete.tts.services.ScreenTextViewModel.<init>(ScreenTextViewModel.kt)
        at com.guillermonegrete.tts.services.ScreenTextService.onStartCommand(ScreenTextService.java:486)
        at android.app.ActivityThread.handleServiceArgs(ActivityThread.java:3770)
        at android.app.ActivityThread.-wrap23(ActivityThread.java) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1764) 
        at android.os.Handler.dispatchMessage(Handler.java:102) 
        at android.os.Looper.loop(Looper.java:154) 
        at android.app.ActivityThread.main(ActivityThread.java:6816) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1563) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1451) 
```

Seems like `getResources().getStringArray(R.array.googleTranslateLanguagesValue)` in `ScreenTextService` is returning a null value.

--- a/app/src/main/java/com/guillermonegrete/tts/services/ScreenTextService.java
+++ b/app/src/main/java/com/guillermonegrete/tts/services/ScreenTextService.java
@@ -60,6 +60,9 @@ import com.guillermonegrete.tts.main.domain.interactors.GetLangAndTranslation;
 
 import javax.inject.Inject;
 
+import dagger.hilt.android.AndroidEntryPoint;
+
+@AndroidEntryPoint
 public class ScreenTextService extends Service {
 
     private WindowManager windowManager;

